bundles/af-core/src/test/resources/form/fileinputv2/exporter-fileinput-datalayer.json

```diff
@@ -24,7 +24,8 @@
       "dorExclusion": false
     },
     "fd:path": "/content/fileinput-datalayer",
-    "dragDropText":"Drag and Drop To Upload"
+    "dragDropText":"Drag and Drop To Upload",
```

Comment:
if we are creating v3, then it should have been `fd:dragDropText`

Reply:
@nit23uec it was introduced in v2, if we will change this, it will cause backward incompatibility..

Reply:
But v3 is created because we are already breaking compatability. Isn't it?

Reply:
Implemented!